build/tfs/common/publish.ts

```diff
@@ -131,9 +131,11 @@ async function doesAssetExist(blobService: azure.BlobService, quality: string, b
 }
 
 async function uploadBlob(blobService: azure.BlobService, quality: string, blobName: string, file: string): Promise<void> {
-	const blobOptions = {
-		contentType: mime.lookup(file),
-		cacheControl: 'max-age=31536000, public'
+	const blobOptions: azure.BlobService.CreateBlockBlobRequestOptions = {
+		contentSettings: {
```

Comment:
I believe this is the correct API for current version of the azure bindings: https://github.com/Azure/azure-storage-node/blob/master/typings/azure-storage/azure-storage.d.ts#L2778

Not sure if the properties were being applied correctly before or just being ignored